deepchecks/vision/checks/performance/robustness_report.py

```diff
@@ -419,3 +396,57 @@ def get_random_image_pairs_from_dataset(original_dataset: VisionData,
 
     # Sort by diff but return only the tuple
     return [s for s, _ in sorted(zip(samples, sort_value), key=lambda pair: pair[1])]
+
+
+HTML_TEMPLATE = """
+<style>
+    .container {{
+        overflow-x: auto;
+        display: flex;
+        flex-direction: column;
+        gap: 10px;
+    }}
+
+    .row {{
+      display: flex;
+      flex-direction: row;
+      align-items: center;    
+    }}
+    
+    .item {{
+      flex: 1;
+      min-width: 200px;
+      position: relative;
+      word-wrap: break-word;
+    }}
+    
+    .image-div {{
+      min-height: 200px;
+    }}
+    
+    .class-div {{
+      text-align: center;
+    }}
+    
+    h3,h4 {{
+        font-family: "Open Sans", verdana, arial, sans-serif;
+        color: #2a3f5f
+    }}
+    
+</style>
+<h3>Augmentation "{aug_name}" Examples</h3>
+<div class="container">
+    <div class="row">
+        <h4 class="item">Class</h4>
+        {class_names}    
+    </div>
+    <div class="row">
+        <h4 class="item">Base Image</h4>
+        {base_images}
+    </div>
+    <div class="row">
+        <h4 class="item">Augmented Image</h4>
+        {aug_images}
+    </div>
+</div>
+"""
```

Comment:
Maybe in a different file?

Reply:
I don't know it's kinda annoying to handle files, and the check is not alone in folder, it's with more performance checks